deps/src/polymake_pairs.cpp

```diff
@@ -0,0 +1,45 @@
+#include "polymake_includes.h"
+
+#include "polymake_tools.h"
+
+#include "polymake_functions.h"
+
+#include "polymake_type_modules.h"
+
+
+void polymake_module_add_pairs(jlcxx::Module& polymake)
+{
+
+    auto type = polymake
+        .add_type<jlcxx::Parametric<jlcxx::TypeVar<1>, jlcxx::TypeVar<2>>>(
+            "StdPair", jlcxx::julia_type("Any", "Base" ));
+
+        type.apply<std::pair<pm::Int,pm::Int>>([&polymake](auto wrapped) {
+            typedef typename decltype(wrapped)::type WrappedT;
+
+            wrapped.template constructor();
+            wrapped.template constructor<int64_t, int64_t>();
+
+            //Pattern to overwrite function in Base
+            polymake.set_override_module(jl_base_module);
+
+            wrapped.method("first", [](WrappedT& P) {
```

Comment:
```suggestion
            wrapped.method("first", [](const WrappedT& P) {
```
It seems I missed one const, or is it possible to use  this for assignment?